xla/service/gpu/transforms/priority_fusion.cc

```diff
@@ -352,6 +353,48 @@ class PriorityFusionQueue {
     fusion_info_cache_.Invalidate(instruction);
   }
 
+  void UpdateRuntimes(
+      GpuPerformanceModel::RunTimes& runtimes, const HloInstruction* consumer,
+      const absl::flat_hash_map<const HloInstruction*, absl::Duration>&
+          original_consumers) {
+    auto it = original_consumers.find(consumer);
+    if (it != original_consumers.end()) {
+      runtimes.time_fused += it->second;
+      auto consumer_cache_result = gpu_performance_model_cache_.Get(*consumer);
+      CHECK(consumer_cache_result.has_value());
+      runtimes.time_unfused += (*consumer_cache_result).exec_time;
+    }
+  }
+
+  // Prepare for incremental updates
+  void ComputeRuntimesOfRemovedConsumers() {
+    for (auto pair : operands_to_new_consumers_) {
+      auto operand = pair.first;
+      auto reverse_it = reverse_map_.find(operand);
+      if (reverse_it == reverse_map_.end()) {
+        continue;
+      }
```

Comment:
If you just want to check if `operand` is in `reverse_map_` then I suggest `reverse_map_.contains(operand)`.